src/component/Book.js

```diff
@@ -1,19 +1,48 @@
 import './styles/book.css';
+import React, { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/book/booksSlice';
+import { deleteBooks, fetchBooks } from '../redux/book/booksSlice';
 
 export default function Book(prop) {
   const {
-    category, title, author, item_id,
+    title, author, item_id, category,
   } = prop;
   const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchBooks());
+  }, []);
   return (
     <li className="list-item">
-      <div>
-        <div>{category}</div>
-        <div>{title}</div>
-        <div>{author}</div>
-        <button type="button" onClick={() => { dispatch(removeBook(item_id)); }}>Remove</button>
+      <div key={item_id} className="first">
+        <div className="book">
+          <p className="category">{category}</p>
+          <p className="b-title">{title}</p>
+          <p className="b-author">{author}</p>
+        </div>
+        <div className="buttongroup">
+          <button type="button">Comment</button>
+          <button className="remove" type="button" onClick={() => dispatch(deleteBooks(item_id))}>Remove</button>
+          <button type="button">Edit</button>
+        </div>
+      </div>
+      <div className="total">
+        <div className="second">
+          <div className="percent">
+            <div className="progress-bar" />
+            <div className="complete">
+              <p className="value">64%</p>
+              <p className="completed">Completed</p>
+            </div>
+          </div>
+        </div>
+        <div className="vertical" />
+        <div className="third">
+          <div>
+            <p className="current-chapter">Current Chapter</p>
+            <p className="num-chapter">Chapter 17</p>
+          </div>
+          <button type="button" className="update">UPDATE PROGRESS</button>
+        </div>
```

Comment:
👍 Great job on your project! However, there's a layout alignment issue to address:

- [ ] Horizontal Alignment: The three elements within this `thard` class element should be left-aligned, starting from the same position horizontally. Currently, they don't match the design's alignment. Please consider adjusting the layout to align with the design specifications.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/MayPyone/Bookstore/assets/55840999/cfd4c780-9c75-479f-977f-974f50cbf6bb) | ![image](https://github.com/MayPyone/Bookstore/assets/55840999/efd75588-6a44-4089-b69a-05aec5a1b6bd)|

By ensuring that these elements are left-aligned, you'll bring your project in line with the design and improve its visual consistency. Keep up the fantastic work! 🌟📏